src/tests/ftest/util/pool_security_test_base.py

```diff
@@ -1,5 +1,5 @@
 """
-  (C) Copyright 2020-2023 Intel Corporation.
+  (C) Copyright 2020-2024 Intel Corporation.
```

Comment:
Looks like this file was accidentally updated?

Reply:
Yes. I did not touch this file..

Reply:
To restore this file, you probably need to manually change the date back, then commit with `--no-verify` so the copyright githook doesn't update it